index.css

```diff
@@ -50,16 +50,18 @@ nav img {
 main {
   margin-top: 24.5vw;
   width: 375px;
+  height: fit-content;
 }
 
 .hero {
   background-color: #1c1a19;
-  width: 375px;
+  width: 355px;
```

Comment:
I suggest you fix the padding and margins of your page.  Currently they seem to be margins on your page that are not present in the original figma design. Its part of the project requirements to match these as close as possible.

Reply:
fixed